src/transform.h

```diff
@@ -68,6 +68,8 @@ int transform_validate(struct augeas *aug, struct tree *xfm);
  */
 int transform_load(struct augeas *aug, struct tree *xfm);
 
+int load_file(struct augeas *aug, struct lens *lens, const char *lens_name, char *filename);
```

Comment:
Better yet, is there a way I can generate an `xfm` to pass to `transform_load()`? This would allow to even pass globs.